File: Ex19.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    // Inserção das variáveis

    float a,VI,t,V;

    // Inserção do primeiro valor
    printf("Insira o valor da aceleracao: ");
    scanf("%f",&a);

    
    // Inserção do segundo valor
    printf("Insira a velocidade inicial: ");
    scanf("%f",&VI);
    
    
    // Inserção do terceiro valor
    printf("Insira o tempo de percurso: ");
    scanf("%f",&t);

    V= VI + (a*t);

    if(V<=40){

        printf("Veiculo muito lento");

    }
    else{

        if(V>40 && V<=60){

            printf("Velocidade Permitida");

        }
        else{

            if(V>60 && V<=80){

                printf("Velocidade de Cruzeiro");

            }
            else{

                if(V>80 && V<=120){

                    printf("Veiculo Rapido");
                }
                else{

                    printf("Veiculo muito Rapido");
                
                }


            }
        }

    }
    return 0;
}

File: Ex19.py
aceleracao = float(input("Insira o valor da aceleraçao do veiculo: "))
vi = float(input("Insira a velocidade inicial: "))
t = float(input("Insira o tempo de percurso: "))

vt = vi + (aceleracao*t)

print(f"A velocidade final e de {vt:.2f}.")

if vt <= 40:
    
    print(f"Veiculo Muito Lento")

else:
    
    if vt > 40 and vt <= 60:

        print("Velocidade permitida.")

    elif vt > 60 and vt <= 80:
        print("Velocidade de cruzeiro.")

    elif vt > 80 and vt <=120:
        print("Veiculo rapido")

    else:
        print("veiculo muito rapido.")